cirq-core/cirq/ops/measure_util_test.py

```diff
@@ -103,9 +103,9 @@ def test_measure_single_paulistring():
     with pytest.raises(ValueError, match='should be an instance of cirq.PauliString'):
         _ = cirq.measure_single_paulistring(q)
 
-    # Coefficient != +1
+    # Coefficient != +1 or -1
     with pytest.raises(ValueError, match='must have a coefficient'):
-        _ = cirq.measure_single_paulistring(-ps)
+        _ = cirq.measure_single_paulistring(-2*ps)
```

Comment:
Add the example in the original issue as a test for `cirq.measure_single_paulistring` to show that it now successfully creates the expected `PauliMeasurementGate`

Reply:
Done